app/services/search_service.rb

```diff
@@ -68,26 +66,39 @@ def profiles(search_criteria)
   end
 
   # Search notes for matching strings and package up as a DocResult
-  def textSearch_notes(srchString)
+  def textSearch_notes(srchString, limit = 25)
     sresult = DocList.new
 
-    notes = find_notes(srchString, 10)
-    notes.each do |match|
+    nodes = find_notes(srchString, limit)
+    nodes.each do |match|
       doc = DocResult.fromSearch(match.nid, 'file', match.path, match.title, 'NOTES', 0)
       sresult.addDoc(doc)
     end
 
     sresult
   end
 
+  # Search nodes package up as a DocResult
+  def textSearch_pages(srchString, limit = 25)
```

Comment:
Do we need an equivalent combined nodes function? Could we concatenate the two existing queries somewhere to reduce redundancy?

Reply:
So, we can use this as our unique method/endpoint to search for nodes, right?

```
def find_nodes(input, _limit = 5, order = :default)
    Node.search(query: input, order: order, limit: 5)
        .group(:nid)
        .where('node.status': 1)
        .distinct
  end

Reply:
This looks good. Do you want to add it here?

Reply:
er, to this PR (i didn't mean to this place in the code, to be clear)

Reply:
So on close reading, I don't think we NEED this in this PR, so i vote to merge, and if you think that there is a good use for a generalized `find_nodes` we can do it in a follow-up PR.

Reply:
We don’t have a ‘/search/nodes’ endpoint. We had a find_nodes() method that I remove to use find_pages(). It is true it just search for pages. If we want to search for all type of nodes we just need:

def find_nodes(input, _limit = 5, order = :default)
    Node.search(query: input, order: order, limit: 5)
  end

So, it would be simpler just to call Node.search()?

Reply:
Agreed, simpler.